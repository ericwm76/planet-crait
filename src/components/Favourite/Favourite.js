import React from 'react';

const Favourite = (props) => {
  return (
      <section className='favourite'>
        <h3>Name</h3>
        <p>Homeworld</p>
        <p>Population of Homeworld</p>
        <p>Species</p>
        <div className='related-films'>
          <p>Related Films</p>

        </div>
        <div className='favourite-button'>
        <i className="fal fa-jedi"></i>
        </div>
      </section>
  )
}

export default Favourite ;