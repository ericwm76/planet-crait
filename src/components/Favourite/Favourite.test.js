import React from 'react';
import { shallow } from 'enzyme';
import Favourite from './Favourite';


describe('Favourite', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Favourite />)
  })

  it('should match snapshot with all data being passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
})