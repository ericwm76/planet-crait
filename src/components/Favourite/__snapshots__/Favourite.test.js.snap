// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favourite should match snapshot with all data being passed in correctly 1`] = `
<section
  className="favourite"
>
  <h3>
    Name
  </h3>
  <p>
    Homeworld
  </p>
  <p>
    Population of Homeworld
  </p>
  <p>
    Species
  </p>
  <div
    className="related-films"
  >
    <p>
      Related Films
    </p>
  </div>
  <div
    className="favourite-button"
  >
    <i
      className="fas fa-jedi"
    />
  </div>
</section>
`;
